<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tactile Data Visualizer</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphicsViewForce">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>651</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphicsViewVelocity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>651</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
